Makefile

```
@@ -7,6 +7,7 @@ VPATH = $(PATH)
 KERNEL_IMAGE ?= autonomy/kernel:65ec2e6
```

Just remembered we need the new kernel:
```suggestion
KERNEL_IMAGE ?= autonomy/kernel:0d2dec2
```